moveit_calibration_plugins/CMakeLists.txt

```
@@ -21,6 +21,10 @@ find_package(catkin REQUIRED COMPONENTS
 find_package(Eigen3 REQUIRED)
 find_package(OpenCV REQUIRED imgcodecs aruco)
 
+if("${OpenCV_VERSION_MAJOR}" STREQUAL "3" AND "${OpenCV_VERSION_MINOR}" STREQUAL "2")
```

Why not using [VERSION_EQUAL](https://cmake.org/cmake/help/latest/command/if.html#version-equal)?
```suggestion
if(OpenCV_VERSION VERSION_EQUAL 3.2)
```